integrate-kobiton-travisci.md

```diff
@@ -0,0 +1,178 @@
+  - [4. Fetch session data through REST API](#4-fetch-session-data-through-REST-API)
+  - [5. (Optional) Travis CI customization](#5-optional-travis-ci-customization)
+  - [6. Feedback](#6-feedback)
+
+## Prerequisites
+  - Kobiton account
+  
+  Please visit https://portal.kobiton.com/register to create new account.
+
+## 1. Configure Travis CI with GitHub repository
+
+  Firstly, let's assume you already have an empty GiHub repository for running automation test.
+  
+  This part will guide you to configure Travis CI to integrate with your automation test GitHub repository. If you've already known how to get it done, please skip this step.
+   > For instruction on how to sync GitHub repositories with Travis CI, follow [this guide.](https://docs.travis-ci.com/user/legacy-services-to-github-apps-migration-guide/)
+
+  This documentation will use Node.js as the default language for the automation test script. Therefore, integrating Node.js plugin to Travis CI is required.
+  > If you do not know how to integrate a Node JS app with Travis CI, follow [this tutorial.](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/) 
+
+## 2. Configure automation test script
+  > You can visit [our blog](https://kobiton.com/blog/automation-web-appium-kobiton-nodejs/) for more details on setting up automation test scripts.
+
+  **2.1 Get username and API key**   
+  Go to https://portal.kobiton.com and login to your Kobiton account.
+  - Get username
+    * Select *user icon* -> **"Profile"** (you might find it in the top right corner of the navigation bar)
+  
+    ![](assets/2_kobiton_username.jpg)
+  - Get API key
+    * Select *user icon* -> **"Settings"**
+  
+    ![](assets/2_kobiton_apikey.jpg)
+
+  **2.2 Get desired capabilities**  
+    The desired capabilities need to be added to the automation test script in order tests to be executed on the Kobiton device.
+  * Login to Kobiton. In the top navigation bar, select **"Devices"**.
+    ![](assets/1_device_bar.jpg)
+  * Hover over any device you want to test with and click on the Automation settings button (the gear symbol)   
+  
+    ![](assets/2_get_device.jpg)
+  * You can choose your preferred language, app type. In this guideline, we will choose **"NodeJS"** for *Language* and **"Hybrid/Native from Apps"** for *App type*. Copy the code on the right to prepare for the next step.
+  
+    ![](assets/2_kobiton_device.jpg)
+
+  **2.3 Configure automation test project**
+  
+  Kobiton already provides some samples for automation test, visit [here](https://github.com/kobiton/samples) for reference.
+
+  >In this guideline, we will use the `Node.js` sample (`samples/javascript` folder) as an example.
+
+  Copy the sample to your GitHub repository and edit your `.travis.yml` file like below:
+  ```yml
+  language: node_js
+  node_js:
+    - '7'
+
+  script: npm run android-app-test
+  ```
+  
+  Open `android-app-test.js` file in your repository.
```

Comment:
Open automation test script file in your repository (If don't have create new one as `automation-smoke-test.js`).
You can use our [samples](https://github.com/kobiton/samples) as the referencing